Documentation/ux_principles.md

```diff
@@ -0,0 +1,51 @@
+# Deluge UX Principles
+## What the Deluge Is
+
+The Deluge is a standalone instrument that also works as a controller/sequencer. It has two core features which set it apart from the competition:
+
+* Tracks can be smoothly changed from the internal synths to midi/cv and back again
+* It features a linear arranger 
+
+These features should be maintained and any new features must not impact them
+
+## What the Deluge Isn't
+The deluge is not a computer, and should not feel like you're operating one. Direct complexity and options should be minimized as much as possible, instead allowing the user to create complexity through combining existing features. An example of this is envelope shape - the envelope attack shape is fixed, but through modulating the attack rate with the envelope it can be smoothly morphed from logarithmic to exponential.
+
+## Guidelines
+
+* Favour explicitness 
+    - Parameters describe their action on the waveform, not their result on the final sound
+    - A user who understands synthesis should immediately be able to edit the deluge
+* Favour simplicity
+    - Menus should never be mandatory
```

Comment:
I think this is too strong of a statement. There's plenty of behavior where menus are mandatory. Device settings and community features is an example, but there's also been talk of things like autosampling and batch actions. We can't change the physical labels on the hardware so certain features will simply not be able to be exposed any way other than through menus, and I think that's okay.

Reply:
I think that's fine, by mandatory I mean that users shouldn't feel like they need to use the menus and are missing out if they don't. I'll change it to 'menus should never feel mandatory' to emphasize that it's about the user experience more than anything. 

E.g. right now all sound design parameters are accessible without using the menu system. Conversely the midi/MPE settings do require a menu, but the defaults are fine 90% of the time and there is some logic in the deluge to detect MPE devices and do reasonable changes to accommodate.